### DISCUSSION TITLE
Get the Homebase + Standalone cams in HA

### DISCUSSION THREAD

**Discussion by K00Kai18** [fuatakgun/eufy_security#912] [Q&A]
Hi All,

Quite pretty new here and in HA... Till now I was able to manage what ever I intent to do but I'm stuck on one point for which I wasn't able to find out any solution... Perhaps someone here can help :-).

I've installed the Eufy Security integration and it works well (apart the live streaming but I'll have look later).
I've invested in a Floodlight cam and as you most probably know such cams are standalone and directly connected to the local wifi and not to the homebase wifi.

Therefore I'm wondering how I can get my floodlight into HA together with my homebase?

TIll now I found some threads explaining how to clone an integration but this looks to work only for official integrations. Since Eufy is instealled via HACS it's not working then (I've tried lol).

I also know Eufy is working on the subject to get working the standalone cams with the Homebase to simplify our life (since to date to arm the system I have to arm Homebase and then floodlight.... 1 single button would ease my daily life and I'm trying to achieve via HA.

Any idea how I can handle that ?

**Comment by K00Kai18** [comment]
Sorry just answered my own question 😅

In fact nothing to do with the integration, simply go into the "eufy-security-ws" add-on configuration and add the serial numbers and IPs of you different homebases and stand alone cams:

```
- serial_number: device1_SN
  ip_address: 192.168.0.1
- serial_number: device2_SN
  ip_address: 192.168.0.2
```

**Reply by SFoskett** [reply]
Note for the next person reading this: Add these "bare" as shown above, not with the "stations:" line listed in the documentation
